application/testing/test_watch.sh

```diff
@@ -21,11 +21,7 @@ pid=$!
 
 function cleanup()
 {
-  if [[ -z "${CTEST_F3D_COVERAGE-}" ]]; then
-    kill $pid
-  else
-    xdotool key Escape
```

Comment:
lets remove xdotool and wmctrl from the images then.

Reply:
I agree